clustpy/deep/vade.py

```diff
@@ -230,7 +237,7 @@ def __init__(self, n_clusters: int, embedding_size: int, weights: torch.Tensor =
             variances = torch.ones(n_clusters, embedding_size)
         assert variances.shape == (n_clusters,
                                    embedding_size), "Shape of the initial variances for the Vade_Module must be (n_clusters, embedding_size)"
-        self.p_var = torch.nn.Parameter(torch.tensor(variances), requires_grad=True)
+        self.p_log_var = torch.nn.Parameter(torch.log(torch.tensor(variances)), requires_grad=True)
```

Comment:
use log of initial variances

Reply:
just be careful if you change this here you not to change the subsequent losses as well

Reply:
All losses are calculated with exp(p_log_var). So the new version should be correct